src/main/java/org/uast/uast/lang/java/RawToNodeConverter.java

```diff
@@ -14,12 +14,12 @@
 import org.uast.uast.base.Node;
 
 /**
- * Converter from the format provided by the JavaParser parser
+ * Converter from the format provided by the JavaParser parser ("raw")
  * to the {@link Node} interface.
  *
  * @since 1.0
  */
-public class JavaParserConverter {
+public class RawToNodeConverter {
```

Comment:
Why raw?

Reply:
I propose the following terminology:
"raw" is what came out of the original parser, that is, the raw structure
"draft" - after conversion to Node interface
"unified" – after adaptation

Reply:
So, how would you name AntlrConverter?

Reply:
In my opinion, RawToNodeConverter may be an abstract name for several converters. However, this class is related only to JavaParser.

Reply:
Ok I will rename it


> So, how would you name AntlrConverter?

AntlrToNodeConverter